src/NHibernate/Logging.log4net.cs

```diff
@@ -92,29 +96,31 @@ public class Log4NetLogger : IInternalLogger
 
 		static Log4NetLogger()
 		{
-			IsErrorEnabledDelegate = DelegateHelper.BuildPropertyGetter<bool>(ILogType, "IsErrorEnabled");
-			IsFatalEnabledDelegate = DelegateHelper.BuildPropertyGetter<bool>(ILogType, "IsFatalEnabled");
-			IsDebugEnabledDelegate = DelegateHelper.BuildPropertyGetter<bool>(ILogType, "IsDebugEnabled");
-			IsInfoEnabledDelegate = DelegateHelper.BuildPropertyGetter<bool>(ILogType, "IsInfoEnabled");
-			IsWarnEnabledDelegate = DelegateHelper.BuildPropertyGetter<bool>(ILogType, "IsWarnEnabled");
-			ErrorDelegate = DelegateHelper.BuildAction<object>(ILogType, "Error");
-			ErrorExceptionDelegate = DelegateHelper.BuildAction<object, Exception>(ILogType, "Error");
-			ErrorFormatDelegate = DelegateHelper.BuildAction<string, object[]>(ILogType, "ErrorFormat");
-
-			FatalDelegate = DelegateHelper.BuildAction<object>(ILogType, "Fatal");
-			FatalExceptionDelegate = DelegateHelper.BuildAction<object, Exception>(ILogType, "Fatal");
-
-			DebugDelegate = DelegateHelper.BuildAction<object>(ILogType, "Debug");
-			DebugExceptionDelegate = DelegateHelper.BuildAction<object, Exception>(ILogType, "Debug");
-			DebugFormatDelegate = DelegateHelper.BuildAction<string, object[]>(ILogType, "DebugFormat");
-
-			InfoDelegate = DelegateHelper.BuildAction<object>(ILogType, "Info");
-			InfoExceptionDelegate = DelegateHelper.BuildAction<object, Exception>(ILogType, "Info");
-			InfoFormatDelegate = DelegateHelper.BuildAction<string, object[]>(ILogType, "InfoFormat");
-
-			WarnDelegate = DelegateHelper.BuildAction<object>(ILogType, "Warn");
-			WarnExceptionDelegate = DelegateHelper.BuildAction<object, Exception>(ILogType, "Warn");
-			WarnFormatDelegate = DelegateHelper.BuildAction<string, object[]>(ILogType, "WarnFormat");
+			var iLogType = System.Type.GetType("log4net.ILog, log4net", true);
```

Comment:
Same here. Moreover, there was no need (anymore?) for having the type stored as a field.